crates/ruff_linter/src/rules/flake8_bugbear/rules/useless_expression.rs

```diff
@@ -69,15 +69,11 @@ impl Violation for UselessExpression {
 /// B018
 pub(crate) fn useless_expression(checker: &mut Checker, value: &Expr) {
     // Ignore comparisons, as they're handled by `useless_comparison`.
-    if value.is_compare_expr() {
+    if matches!(value, Expr::Compare(_) | Expr::EllipsisLiteral(_)) {
```

Comment:
By the way, we could flag `EllipsisLiteral` I think? As long as we ignore stub methods...

Reply:
But lets ignore that for this PR.